### PR TITLE
Merge sda-doa

### PR DESCRIPTION
**Pull request by Parisa68**
**Related issue(s) and PR(s)**  
This PR closes issue #857 .

**Description**
Merges sda-doa into this repo.

### CODE REVIEW

**Comment by Parisa68** [comment]
https://github.com/dependabot rebase

File: sda-doa/docker-compose-posix-outbox.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  mockauth:
+    image: cscfi/beacon-python
+    ports:
+      - 8000:8000
+    volumes:
+      - ./test/mock_auth.py:/mock_auth.py
+    entrypoint: ["python", "/mock_auth.py", "0.0.0.0", "8000"]
+
+  private-mq:
+    image: uiobmi/localega-broker-private:latest
```
Comment (jbygdell) [inline sda-doa/docker-compose-posix-outbox.yml:45]:
```suggestion
    build:
      context: ../../rabbitmq
    container_name: rabbitmq
    image: ghcr.io/neicnordic/sensitive-data-archive:PR${PR_NUMBER}-rabbitmq
```

**Review by jbygdell — CHANGES_REQUESTED** [review]
- The entire `.github` folder must be removed so we don't get any conflicts with regards to depenabot and such.
- The relevant parts of the workflows should be merged into the corresponding file of the root of the repo.

- The docker compose files should be moved into `.github/integration` in the root of the repo and be reworked similar to the existing compose files.
 
- Keys, certificates and such should be generated dynamically if needed and not be committed to the repository.
 
- Relevant options for this app should be added to the `Makefile`

**Comment by kjellp** [comment]
Thanks @jbygdell ! We will start resolving as much as possible based on these comments. The plan is to have to PRs, an initial "import" one,  and a second "complete the integration tests " (with main repo root folder ).

**Review by jbygdell — APPROVED** [review]

**Review by kjellp — APPROVED** [review]
LGTM - remaining integrations are carried fwd in #901

**Review by jbygdell — COMMENTED** [review]
@Parisa68 do a `update with rebase` on this repo so it gets matched with main, then it can be merged more easily.

**Pull request merged**